packages/mermaid/src/diagrams/c4/c4Detector.ts

```diff
@@ -1,5 +1,20 @@
-import type { DiagramDetector } from '../../diagram-api/types';
+import type { ExternalDiagramDefinition } from '../../diagram-api/types';
 
-export const c4Detector: DiagramDetector = (txt) => {
+const id = 'c4';
+
+const detector = (txt: string) => {
```

Comment:
It seems the majority of other diagram detectors have a declaration "const detector: DiagramDetector = ...",
but in this file the detector type is not present. It's only declared as "const detector = ..."